frontend/src/components/Candidate-Decider/CandidateDecider.tsx

```diff
@@ -12,17 +12,18 @@ import {
 } from './useCandidateDeciderInstance';
 import Button from '../Common/Button/Button';
 import Input from '../Common/Input/Input';
+import Selector, { RatingOptions } from '../Common/Selector/Selector';
 
 type CandidateDeciderProps = {
   uuid: string;
 };
 
-const ratings = [
+const ratings: RatingOptions[] = [
   { value: 1, text: 'Strong No', color: 'red' },
   { value: 2, text: 'No', color: 'orange' },
   { value: 3, text: 'Maybe', color: 'yellow' },
   { value: 4, text: 'Yes', color: 'green' },
-  { value: 5, text: 'Strong Yes', color: 'green ' },
+  { value: 5, text: 'Strong Yes', color: 'darkgreen' },
   { value: 0, text: 'Undecided', color: 'grey' }
```

Comment:
Nit: since we're in an American setting, we should use American naming conventions lol

```suggestion
  { value: 0, text: 'Undecided', color: 'gray' }
```